### PR TITLE
replace naming of project

### PR DESCRIPTION
**Pull request by CeEv**
# Short Description

## Links to Ticket and related Pull-Requests
- https://ticketsystem.hpi-schul-cloud.org/browse/SC-9114

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: deploy/compose-client_default.dummy [hunk 1]
```diff
@@ -23,7 +23,7 @@ services:
       - NODE_ENV=production
       - TZ=Europe/Berlin
       - LEGACY_CLIENT_URL=http://client:3100
-      - API_URL=https://api.$SYSTEM.schul-cloud.org
+      - API_URL=https://api.$SYSTEM.hpi-schul-cloud.org
```
Comment (Metauriel) [inline deploy/compose-client_default.dummy:26]: please check if .org is correct here

Comment (Metauriel) [reply deploy/compose-client_default.dummy:26]: also in hte other compose files

**Review by Metauriel — CHANGES_REQUESTED** [review]

**Comment by CeEv** [comment]
@schul-cloud-bot update screenshots

File: deploy/compose-client_default.dummy (hunk 1 above)
Comment (CeEv) [reply deploy/compose-client_default.dummy:26]: done werden nicht mehr verwendet.

**Review by Metauriel — APPROVED** [review]

**Pull request merged**